### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by lpsinger**

### CODE REVIEW

**Comment by astrofrog** [comment]
I don't think this is a typo - it is referring to the test function. But maybe we should add () after test?

**Comment by lpsinger** [comment]
Ohhh... I see.

**Comment by lpsinger** [comment]
Take a look at this version.

File: {{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/conftest.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 This file is used to configure the behavior of pytest when using the Astropy
 test infrastructure. It needs to live inside the package in order for it to
 get picked up when running the tests inside an interpreter using
-packagename.test
+`{{ cookiecutter.package_name }}.test()`.
```
Comment (astrofrog) [inline {{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/conftest.py:6]: I think it should be module name instead of package name?

**Review by astrofrog — CHANGES_REQUESTED** [review]

File: {{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/conftest.py (hunk 1 above)
Comment (lpsinger) [reply {{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/conftest.py]: Fixed.

**Review by astrofrog — APPROVED** [review]

**Pull request merged**